setup.py

```diff
@@ -7,42 +7,42 @@
 here = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent
 
 
-def read_reqs( reqs_path: Path):
-    return re.findall(r'(^[^#-][\w]+[-~>=<.\w]+)', reqs_path.read_text(), re.MULTILINE)
+def read_reqs(reqs_path: Path):
```

Comment:
Last week, we created a new common structure for the ``setup.py``. It is not too far from this one so it should be very easy to adapt. See the section in https://github.com/ITISFoundation/osparc-simcore/pull/2669  and checkout some of the ``setup.py`` files in the changes section